draft-fossati-tls-attestation.md

```diff
@@ -1077,6 +1077,9 @@ TBD.
 
 # Privacy Considerations {#priv-cons}
 
+In this section, we are assuming that the Attester is a TLS client, representing an individual person.
+We are concerned about the potential leakage of unnecessary information about that person, such as the correlation of different connections initiated by them.
+
```

Comment:
I think, the statement on line 1083, is regardless of what mode the Attestation is carried out. Whether in BG Check or PP Model, the Verifier has access to detailed information about the Attester's TCB. 

Shall we not just remove which Mode we are operating in and just state that `Verifier gets detailed information`

Hence the following solutions apply to preserve anonymity..?

Reply:
> I think, the statement on line 1083, is regardless of what mode the Attestation is carried out. Whether in BG Check or PP Model, the Verifier has access to detailed information about the Attester's TCB.
> Shall we not just remove which Mode we are operating in and just state that `Verifier gets detailed information`
> 
> Hence the following solutions apply to preserve anonymity..?

No, the privacy problem we are trying to highlight here is the potential for correlating the TLS peers' identities and the time of the secure channel establishment by the Verifier.